zerver/openapi/javascript_examples.js

```diff
@@ -62,6 +62,30 @@ const {main, add_example} = ExamplesHandler();
 
 // Declare all the examples below.
 
+add_example('send_message', '/messages:post', 200, async (client) => {
+    // {code_example|start}
+    // Send a stream message
+    let params = {
+        to: 'Denmark',
+        type: 'stream',
+        topic: 'Castle',
+        content: 'I come not, friends, to steal away your hearts.',
+    };
+    const result_1 = await client.messages.send(params);
+
+    // Send a private message
+    const user_id = 9;
+    params = {
+        to: [user_id],
+        type: 'private',
+        content: 'With mirth and laughter let old wrinkles come.',
+    };
+    const result_2 = await client.messages.send(params);
+
+    return [result_1, result_2];
```

Comment:
So, I think this would be a lot better as 2 semi-independent examples of how to send a stream message and send a private message, like they were in the original implementation.  

The `result_1` and `result_2` variables just don't look like professional documentation, and we're losing the ability to easily copy-paste the simple "send a message to a stream" example, which is likely the only thing 95% of users will want to do with this example or the `zulip-js` API overally.

I think rather than this:

```
const config = { zuliprc: 'zuliprc' };

Zulip(config).then(async (client) => {
    // Send a stream message
    let params = {
        to: 'Denmark',
        type: 'stream',
        topic: 'Castle',
        content: 'I come not, friends, to steal away your hearts.',
    };
    const result_1 = await client.messages.send(params);

    // Send a private message
    const user_id = 9;
    params = {
        to: [user_id],
        type: 'private',
        content: 'With mirth and laughter let old wrinkles come.',
    };
    const result_2 = await client.messages.send(params);

    return [result_1, result_2];
}).then(console.log).catch(console.err);
```

we want the output to be more like this:

```
const zulip_client = Zulip({ zuliprc: 'zuliprc' });

// Send a stream message
zulip_client.then(async (client) => {
    let params = {
        to: 'Denmark',
        type: 'stream',
        topic: 'Castle',
        content: 'I come not, friends, to steal away your hearts.',
    };
    return await client.messages.send(params);
}).then(console.log).catch(console.err);;

// Send a private message
zulip_client.then(async (client) => {
    const recipient_user_id = 9;
    params = {
        to: [recipient_user_id],
        type: 'private',
        content: 'With mirth and laughter let old wrinkles come.',
    };
    return await client.messages.send(params);
}).then(console.log).catch(console.err);
```